staging/src/k8s.io/apiserver/pkg/util/webhook/authentication.go

```diff
@@ -196,6 +196,9 @@ func restConfigFromKubeconfig(configAuthInfo *clientcmdapi.AuthInfo) (*rest.Conf
 		config.Username = configAuthInfo.Username
 		config.Password = configAuthInfo.Password
 	}
+	if configAuthInfo.Exec != nil {
```

Comment:
any other webhooks need this as well? this was likely to have been copied/pasted

Reply:
This looks to be the only spot this is copy pasted. It looks like it was copy pasted from client-go. I'll double check that this works as expected.